tests/decorators/test_uri_parsing.py

```diff
@@ -1,5 +1,6 @@
-import pytest
 from werkzeug.datastructures import MultiDict
+
```

Comment:
Unfortunately this breaks the isort step in the build. See the end of the python 3.6 travis run for details.